docs/api/I13nUtils.md

```diff
@@ -0,0 +1,40 @@
+## I13nUtils
+
+The `I13Utils` mixin provides util functions for you to access the i13n nodes and execute i13n functionalities
```

Comment:
I think its confusing to have I13Utils and I13nMixin, its not clear when to use which one. Maybe just merge them and always use I13nMixin?